src/figuresClasses.ts

```diff
@@ -1,19 +1,96 @@
-export interface Figure {
+// import { moveEmitHelpers } from 'typescript';
+type Color = 'red' | 'green' | 'blue';
 
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+export interface Figure {
+  color: Color;
+  shape: Shape;
+  getArea: () => number;
 }
 
+const roundResult = (num: number): number => {
+  return Math.floor(num * 100) / 100;
+};
+
+const checkParameters = (...parameters: number[]): void => {
+  if (parameters.some((num) => num <= 0)) {
+    throw new Error('Enter number bigger than 0!');
+  }
+};
+
 export class Triangle {
```

Comment:
actually it would be better to explicitly mention what interface we implement here.
```suggestion
export class Triangle implements Figure {
```
It is used to verify that a class conforms to a specific interface